### PR TITLE
Add geo interface in native client.

### PR DESCRIPTION
**Pull request by CPWstatic**
Seems not required anymore.

### CODE REVIEW

**Comment by CPWstatic** [comment]
Jenkins go!

File: src/tools/native-client/pom.xml [hunk 1]
```diff
@@ -35,6 +35,11 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (darionyaphet) [inline src/tools/native-client/pom.xml]: why don't move `S2` into `Spark Generator` ？

**Comment by darionyaphet** [comment]
`native client` is a lightly warpper for cpp interface. The `dimension transformation` seems not matching

**Comment by CPWstatic** [comment]
> Could we change the class name?
> native_client is not reasonable. "NabulaUtils" maybe better.

Agree.

**Comment by dangleptr** [comment]
Do we still need it?  @CPWstatic

**Comment by darionyaphet** [comment]
Maybe we should support this method in `nebula java` ?

**Comment by dangleptr** [comment]
> Maybe we should support this method in `nebula java` ?

Sounds good. Let's reimplement it in nebula-java.  Close it.  @darionyaphet

**Pull request closed without merging**